docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,489 @@
+  
+# Dapr 0.11.0
+
+It's been seven weeks, and We're happy to announce the release of Dapr v0.11.0!
```

Comment:
We are happy to announce yet another release of Dapr. With this v0.11 release we....

Reply:
Agree. We can drop the "It's been seven days". Starting from this week we'll include a blog post for each release which can include this kind of tone